packages/@okta/vuepress-site/docs/guides/refresh-tokens/get-refresh-token/index.md

```diff
@@ -20,32 +20,86 @@ In the case of the authorization code flow, you use the authorization server's `
 
-For example, with the `password` grant type you can include an `openid` scope alongside the `offline_access` scope:
+#### Example request
 
-```BASH
-POST https://${yourOktaDomain}/oauth2/default/v1/token
+With the `password` grant type, you can include an `openid` scope alongside the `offline_access` scope to also get back an ID token.
 
-grant_type=password
+```BASH
+POST https://${yourOktaDomain}/oauth2/default/v1/token?grant_type=password
  &redirect_uri=http%3A%2F%2Flocalhost%3A8080
  &username=example%40mailinator.com
  &password=a.gReAt.pasSword
  &scope=openid%20offline_access
 ```
 
-You would then get back an ID token alongside your access and refresh tokens. See the [Okta OAuth 2.0 reference page](/docs/reference/api/oidc/#response-properties).
+#### Example response
+
+You would then get back an ID token as well as your access and refresh tokens. See the [Okta OAuth 2.0 reference page](/docs/reference/api/oidc/#response-properties).
+
+> **Note:** The access and ID tokens are truncated for brevity.
+
+```JSON
+{
+    "token_type": "Bearer",
+    "expires_in": 3600,
+    "access_token": "eyJraWQi.....T2aA5ottg",
+    "scope": "offline_access openid",
+    "refresh_token": "cBMrwDsXRwPqVmCQx7I5IX0jQ9-Lc_zHOgYeab1xZm4",
+    "id_token": "eyJra.....ezAriw"
+}
+```
 
 ### Get a new access token/ID token silently for your SPA
 
-In a normal Single-Page Application (SPA), it is usually undesirable to redirect the user to a sign-in page during normal navigation. For example, a user could request access to a resource, prompting your SPA to send a request to the Okta `/authorize` endpoint. Normally, if a user doesn't have a valid session, this request results in a redirection to a sign-in page. To avoid this disruptive redirection, the endpoint allows for a request parameter called `prompt`. If the value of the `prompt` parameter is `none`, this guarantees that the user won't be prompted to sign in, regardless of whether they have an active session. Instead, your application either silently obtains the requested tokens or an OAuth error response. How to act on the error is up to you.
+With a normal Single-Page Application (SPA), it is usually undesirable to redirect the user to a sign-in page during normal navigation. For example, a user could request access to a resource, prompting your SPA to send a request to the Okta `/authorize` endpoint. Normally, if a user doesn't have a valid session, this request results in a redirection to a sign-in page. To avoid this disruptive redirection, the endpoint allows for a request parameter called `prompt`. If the value of the `prompt` parameter is `none`, this guarantees that the user won't be prompted to sign in, regardless of whether they have an active session. Instead, your application either silently obtains the requested tokens or an OAuth error response occurs.
```

Comment:
```suggestion
With a normal Single-Page Application (SPA), it is usually undesirable to redirect the user to a sign-in page during normal navigation. For example, a user could request access to a resource, prompting your SPA to send a request to the Okta `/authorize` endpoint. Normally, if a user doesn't have a valid session, this request results in a redirect to a sign-in page. To avoid this disruptive redirect, the endpoint allows for a request parameter called `prompt`. If the value of the `prompt` parameter is `none`, this guarantees that the user won't be prompted to sign in, regardless of whether they have an active session. Instead, your application either silently obtains the requested tokens or an OAuth error response occurs.
```